app/components/submission-status-cell.js

```diff
@@ -1,43 +1,32 @@
 import Component from '@ember/component';
 
 export default Component.extend({
-  store: Ember.inject.service(),
-  repoCopies: null,
-
-  init() {
-    this._super(...arguments);
-    // Query to get all 'repositoryCopy' objects that are associated with the 'publication'
-    // in this record's 'submission'
-    this.get('store').query('repositoryCopy', {
-      size: 500,
-      from: 0,
-      query: {
-        term: { publication: this.get('record.publication.id') }
-      }
-    }).then(rc => this.set('repoCopies', rc));
-  },
   // For specs: https://github.com/OA-PASS/pass-ember/issues/568
-  status: Ember.computed('record', 'repoCopies', function () {
+  status: Ember.computed('record', function () {
     const submission = this.get('record');
-    const repoCopies = this.get('repoCopies');
+    const repoCopies = this.get('column.repoCopiesMap')[submission.get('id')];
+    const deposits = this.get('column.depositsMap')[submission.get('id')];
 
-    if (!repoCopies) { // this will be NULL/UNDEFINED until search returns
-      return '';
+    if (repoCopies.any(rc => rc.get('copyStatus') == 'stalled')) {
+        return 'Stalled';
     }
 
-    if (repoCopies.get('length') === 0) {
-      const source = submission.get('source');
-      if (source == 'pass' && submission.get('submitted')) {
-        return 'Submitted';
-      } else if (source == 'other' && !submission.get('submitted')) {
+    if (submission.get('source') == 'other' && !submission.get('submitted')) {
         return 'Manuscript expected';
+    }
+
+    if (repoCopies.get('length') < deposits.get('length')) {
+      if (submission.get('source') == 'pass') {
+        return 'Submitted';
```

Comment:
Submitted should appear when there are zero RepositoryCopies for the Submission, `Submission.source=pass` and `Submission.submitted=true`. Currently these are appearing as more details links.